.github/workflows/_deploy_kafka_test.yml

```diff
@@ -66,12 +66,20 @@ jobs:
           SIDECAR_IMAGE: ${{ inputs.key-release-image }}
           CONSUMER_IMAGE: ${{ inputs.consumer-image }}
           PRODUCER_IMAGE: ${{ inputs.producer-image }}
+          EVENT_HUB_NAMESPACE: ${{ vars.EVENT_HUB_NAMESPACE }}
         id: run-workload
         run: | 
+
+          echo "Getting Event Hub connection env vars..."
+          result=$(az eventhubs namespace authorization-rule list --resource-group $RESOURCE_GROUP --namespace-name $EVENT_HUB_NAMESPACE)
+          export EVENT_HUB_AUTH_RULE_NAME=$(echo "$result" | jq -r '.[0].name')
+          echo $EVENT_HUB_AUTH_RULE_NAME
+          result=$(az eventhubs namespace authorization-rule keys list --resource-group $RESOURCE_GROUP --namespace-name $EVENT_HUB_NAMESPACE --name $EVENT_HUB_AUTH_RULE_NAME)
+          export EVENTHUB_CONNECTION_STRING="$(echo "$result" | jq -r '.primaryConnectionString')"
+          echo $EVENTHUB_CONNECTION_STRING
```

Comment:
Doesn't the connection string contain credentials?  Probably should not routinely print this.